.github/workflows/publish_docker_images_cron.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Clone tools branch
-        run: git clone -b v0.8.1 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b pg-14-docker --depth=1  https://github.com/citusdata/tools.git tools
```

Comment:
After approval of https://github.com/citusdata/tools/pull/182 and tag creation, branch name will be replaced

Reply:
Please replace the tag name, and update the line here.

Reply:
Fixed